src/main/java/edu/oregonstate/cs361/battleship/BattleshipModel.java

```diff
@@ -129,14 +130,16 @@ public void shootAtComputer(int row, int col) {
     }
 
     public void shootAtPlayer() {
-        int max = 10;
-        int min = 1;
-        Random random = new Random();
-        int randRow = random.nextInt(max - min + 1) + min;
-        int randCol = random.nextInt(max - min + 1) + min;
-
-        Coordinate coor = new Coordinate(randRow,randCol);
```

Comment:
Comment out lines instead of deleting them next time. We could use this part to implement the hard difficulty.